integration_tests/gdk/components/test_integ_PublishCommand.py

```diff
@@ -119,6 +119,108 @@ def test_publish_run_with_bucket_argument(mocker, get_service_clients, mock_proj
     assert spy_get_caller_identity.call_count == 1  # Get account number
 
 
+def test_publish_run_with_region_and_options_argument(mocker, get_service_clients, mock_project_config):
+    mock_build_dir_exists = mocker.patch(
```

Comment:
We shouldn't be mocking all this dependencies, this just couples the tests to implementation details and will break the tests down the line if we change the implementation